danger-kotlin-library/src/main/kotlin/systems/danger/kotlin/models/danger/DangerResults.kt

```diff
@@ -1,14 +1,14 @@
 @file:UseSerializers(ViolationSerializer::class)
 
-package systems.danger.kotlin
+package systems.danger.kotlin.models.danger
 
 import kotlinx.serialization.Serializable
 import kotlinx.serialization.UseSerializers
 import systems.danger.kotlin.sdk.Violation
-import systems.danger.kotlin.serializers.ViolationSerializer
+import systems.danger.kotlin.models.serializers.ViolationSerializer
 
 @Serializable
-data class Meta(
+internal data class Meta(
```

Comment:
Meta as well shouldn't be visible from the Danger File as well, changed to internal